### ISSUE TITLE
Windows Script Host - "Can not find script file"

### ISSUE THREAD

**Issue by milutii** [ThisIsWin11/ThisIsWin11#1]
Hi there,

I just installed ThisIsWin11 this afternoon, and ever since I've been receiving two different script issues (see image).  I've received each one at least 5 times, I'd say.  Any advice?  I have limited background/education so please feel free to dumb it down a little bit. ;) hahaha.

![windows-script-host](https://user-images.githubusercontent.com/112460322/209586888-c637cc02-4741-47d1-acd2-39e6f504d8d1.jpg)

**Comment by SkyFoxCoder** [comment]
Do NOT use this, this is a fork of the real "ThisIsWin11" that contains a Crypto Miner, I suggest running a virus scan because you are definitly infected by it. The original version is here: https://github.com/builtbybel/ThisIsWin11
